doc/src/modules/polys/ringseries.rst

```diff
@@ -0,0 +1,154 @@
+.. _polys-ringseries:
+
+=====================================
+Series Manipulation using Polynomials
+=====================================
+
+Any finite Taylor series, for all practical purposes is, in fact a polynomial.
+This module makes use of the efficient representation and operations of sparse
+polynomials for very fast multivariate series manipulations. Typical speedups
+compared to SymPy's ``series`` method are in the range 20-100, with the gap
+widening as the series being handled gets larger.
+
+All the functions expand any given series on some ring specified by the user.
+Thus, the coefficients of the calculated series depend on the ring being used.
+For example::
+
+    >>> R, x, y = ring('x, y', QQ)
+    >>> rs_sin(x*y, x, 5)
+    -1/6*x**3*y**3 + x*y
+
+``QQ`` stands for the Rational domain. Here all coefficients are rationals. It
+is recommended to use ``QQ`` with ring series as it automatically chooses the
+fastest Rational type.
+
+Similarly, if a Real domain is used::
+
+    >>> R, x, y = ring('x, y', RR)
+    >>> rs_sin(x*y, x, 5)
+    -0.166666666666667*x**3*y**3 + x*y
+
+Though the definition of a polynomial limits the use of Polynomial module to
+Taylor series, we extend it to allow Laurent and even Puiseux series (with
+fractional exponents)::
+
+    >>> R, x, y = ring('x, y', QQ)
+    >>> rs_cos(x**QQ(1, 3) + x*y, x, 2)
+    -x**(4/3)*y + 1/24*x**(4/3) - 1/2*x**(2/3) + 1
+
+    >>> rs_tan(x**QQ(2, 5)*y**QQ(1, 2), x, 2)
+    1/3*x**(6/5)*y**(3/2) + x**(2/5)*y**(1/2)
```

Comment:
Thanks for adding this section. Since the polynomials do not allow fractional exponents, how is this achieved to work? Did we modify the polys module in the end (and if so how)?

Also, I think both examples are Puiseux series. To do Laurent series, you can expand something like `sin(x)/x**4`:
```
In [5]: series(sin(x)/x**4, x, 0, 6)
Out[5]: x**(-3) - 1/(6*x) + x/120 - x**3/5040 + x**5/362880 + O(x**6)
```
I.e. I guess you need to first use rs_sin, and then divide by x^4. This would also be a nice example to show how this is done using the low level methods.